specs/AgriFood/AgriParcelOperation/example-normalized-ld.jsonld

```diff
@@ -0,0 +1,67 @@
+{
+    "@context": [
+        "http://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld",
+        "https://raw.githubusercontent.com/GSMADeveloper/NGSI-LD-Entities/master/examples/Agri-Parcel-Operation-context.jsonld"
+    ],
+    "id": "urn:ngsi-ld:AgriParcelOperation:e1e9d3a3-074f-46f1-9375-52000d05a62b",
+    "type": "AgriParcelOperation",
+    "createdAt": "2017-01-01T01:20:00Z",
+    "modifiedAt": "2017-05-04T12:30:00Z",
+    "agriParcel": {
+        "type": "Relationship",
+        "object": "urn:ngsi-ld:AgriParcel:318366a9-7643-4d8e-9a11-c76a8c29d8eb"
+    },
+    "operationType": {
+        "type": "Property",
+        "value": "fertiliser"
+    },
+    "description": {
+        "type": "Property",
+        "value": "Monthly fertiliser application"
+    },
+    "result": {
+        "type": "Property",
+        "value": "ok"
+    },
+    "plannedStartAt": {
+        "type": "DateTime",
```

Comment:
that's not the right encoding in LD, please check https://github.com/FIWARE/dataModels/blob/master/specs/ngsi-ld_howto.md#airquality-in-ngsi-ld-format